triton-vm/src/table/master_table.rs

```diff
@@ -2069,4 +2071,115 @@ mod tests {
         let expected_digest = Tip5::hash_varlen(&elements);
         prop_assert_eq!(expected_digest, pending_absorb_digest);
     }
+
+    /// Test whether the AIR constraint evaluators are the same between
+    ///  (a) the time when this test was written or last updated; and
+    ///  (b) the time when the test is being executed.
+    ///
+    /// This test catches (whp) unintended changes, whether due to
+    /// nondeterminisms (on a single machine or across various machines) or due
+    /// to changes to the definitions of the constraints. If the change to the
+    /// constraints was intentional, this test should fail until the expected
+    /// value in the assert-statement is updated.
+    #[test]
+    fn air_constraints_evaluators_have_not_changed() {
+        let seed: [u8; 32] = [
+            193, 123, 173, 34, 45, 172, 73, 224, 156, 208, 121, 29, 245, 142, 215, 124, 188, 114,
+            203, 148, 186, 245, 224, 76, 142, 129, 82, 225, 81, 28, 216, 183,
+        ];
+
+        let mut rng: StdRng = SeedableRng::from_seed(seed);
```

Comment:
The given seed was produced by `thread_rng()` and therefore guaranteed to be random. In contrast, `0` doesn't seem very random. Are you sure you produced it from `thread_rng()` or something similar?

Moreover, applying this suggestion causes the test to fail, because the expected value comes from a different seed.

Reply:
For one isolated test, there is no practical difference and your suggestion is simpler.

In the context of many derandomized probabilistic tests, you reduce the quality of the entire test suite by always using the same seed. You want your tests to be independent of each other and always sampling the same indeterminate (for instance) correlates them. 

In light of that consideration, I would suggest that it is good practice to always use independent random seeds in derandomized probabilistic tests. (Even if that complicates the code, and even if it is only one test case.)

Reply:
Those arguments are sound. They do not preclude using `let mut rng = StdRng::seed_from_u64(/* independent random seed */);`